packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -100,6 +101,7 @@ export function SignIn() {
 
 SignIn.Header = (): JSX.Element => null;
 SignIn.Footer = () => {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Let's not disable rules of hooks here. There are a handful of rules of hooks violations in Authenticator which need to be dealt with and I don't want to have to find them manually.